lib/facter/choco_installpath.rb

```diff
@@ -0,0 +1,13 @@
+Facter.add('choco_installpath') do
+  confine :kernel => 'windows'
+
+  setcode do
+    require 'win32/registry'
+
+    value = 'C:/ProgramData/Chocolatey'
+    Win32::Registry::HKEY_LOCAL_MACHINE.open('SYSTEM\CurrentControlSet\Control\Session Manager\Environment') do |reg|
+      value = reg['ChocolateyInstall'] if reg.has_key? 'ChocolateyInstall'
```

Comment:
Doesn't look like it does. But it does have keys and each_key enumeration:
* http://ruby-doc.org/stdlib-1.9.3/libdoc/dl/rdoc/Win32/Registry.html#method-i-keys
 * https://github.com/ruby/ruby/blob/v1_9_3_551/ext/dl/win32/lib/win32/registry.rb#L547-L568
* https://github.com/ruby/ruby/blob/v2_1_6/ext/win32/lib/win32/registry.rb#L590-L612